### ISSUE TITLE
Source code license?

### ISSUE THREAD

**Issue by vatterspun** [mobzystems/MOBZync#1]
I appreciate you sharing the source code but could you give it license?  I know a familiar license has attracted some attention for similar projects.

Thanks!

**Comment by mobzystems** [comment]
Could you elaborate on that? The license is supposed to mean "absolutely free, no strings attached" with maybe a mention somewhere, but that's it. What license do we need then, in your opionion?

**Comment by mobzystems** [comment]
Oops - did not mean to close the issue! Reopening...

**Comment by vatterspun** [comment]
First, I'm not a lawyer just someone that works with a lot of open source devs.

> Could you elaborate on that? The license is supposed to mean "absolutely free, no strings attached" with maybe a mention somewhere, but that's it. What license do we need then, in your opionion?

I know it sounds stupid (and frankly it is) but free / public domain / etc. just don't mean what they used to.  Part of this is the fault of the US with its patent and copyright law, but part of it is just people doing any number of variants on sort-of-free software.  Anyway the short version of my advice is to use a license that's widely used (so other sources would come to your aid if a legal issue came up) and at least *mentions* patent law.

Apache 2.0 is probably the closest to "do whatever you want to but don't pull some kind of patent BS" license.  It's understood as compatible with many other licenses so your code is more likely to find it's way into other projects.

Thanks for hearing me out.  I get that this stuff is boring.

If you're curious, I recommend starting with the Wikipedia post on the madness surrounding public domain: https://en.wikipedia.org/wiki/Public_domain#Public_domain_like_licenses and checking out the Apache license in plain english from TLDR legal: https://tldrlegal.com/license/apache-license-2.0-%28apache-2.0%29
